docs/daemon/install.mdx

```diff
@@ -7,11 +7,30 @@ description: How to install the Kwil Daemon tool
 slug: /daemon/installation
 ---
 
-While Kwil is under heavy development, the team has made the project closed source. However, the Kwil team regularly releases
-Kwil Docker images to Docker Hub. This allows you to run Kwil on your own machine, without having to build the project from source.
+Until the main Kwil DB repository is open source, the `kwild` command line 
+application may be downloaded from the Kwil binary release repository.
 
-To install the most recent Kwil image, run:
+
+## Download the `kwild` Executable
+
+The Kwil daemon(`kwild`) will be packaged together with Kwil admin tool(`kwil-admin`) and Kwil cli(`kwil-cli`).
```

Comment:
Nit, but when parenthesizing, a space between the preceding word/phrase and the parenthesis the norm.